### DISCUSSION TITLE
Not authorised with basic example request

### DISCUSSION THREAD

**Discussion by willh36** [transitland/transitland#422] [Q&A] [answered]
Hi, I'm trying to use the v2 REST API for a school project and got access to the academic/hobbyist plan a month ago. I originally made my account a couple of months ago and tried this command: `https://transit.land/api/v2/feeds?api_key=MY-API-KEY-IS-HERE` which worked perfectly. However, I tried this command more recently and got the following response: `{"error":"not authorized"}`. I don't believe I've hit any rate limit since I haven't used the API in many months and when I did I only made about 30 requests in total. I've tried making a different account and then using the API key from that account and still get the same response. I'm really confused and need to have this working really soon for a fast-approaching deadline so I'd really appreciate any help at all :)

**Comment by irees** [comment, marked as answer]
Hi - can you try using `https://transit.land/api/v2/rest/feeds` as the endpoint?

**Reply by willh36** [reply]
Yes that works! Thank you for your help and sorry for the stupid question
